### PR TITLE
Add api key statement to readme for consistency. 

### PR DESCRIPTION
**Pull request by HarleyKwyn**
Remove unused fn in example for batch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,16 +106,7 @@ indico
...
-
-function fn(err, res) {
-  if (err) {
-    console.log('err: ', err);
-    return;
-  }
-
-  console.log(res):
-}
+var indico = require('indico.io')f
```
Comment (madisonmay) [inline README.md]: There's an awkward `f` hanging out at the end of this line, haha.

Comment (HarleyKwyn) [inline README.md]: Omg so rude. I'll fix that.

**Comment by madisonmay** [comment]
Awesome, thanks for the PR!

**Comment by HarleyKwyn** [comment]
np :)

**Pull request merged**